src/components/ConnectionFlowDialogs.tsx

```diff
@@ -134,12 +81,27 @@ const ConnectionDialogs = () => {
           onBackClick={actions.onBackClick}
           onNextClick={actions.onNextClick}
           config={config}
-          onSkip={onSkip}
+          onSkip={() => actions.setFlowStep(ConnectionFlowStep.ConnectBattery)}
           onSwitch={actions.switchFlowType}
         />
       );
     }
     case ConnectionFlowStep.WebUsbFlashingTutorial: {
+      const connectAndFlash = async () => {
+        const onFlashSuccess = (newStage: ConnectionStage) => {
+          // Inferring microbit name saves the user from entering the pattern
+          // for bluetooth connection flow
+          if (newStage.bluetoothMicrobitName) {
+            setMicrobitName(newStage.bluetoothMicrobitName);
+          }
+        };
+
+        logging.event({
+          type: "connect-user",
+          message: "radio-bridge",
+        });
```

Comment:
We only want this event for radio-bridge.
```suggestion
        if (stage.flowType === ConnectionFlowType.ConnectRadioBridge) {
          logging.event({
            type: "connect-user",
            message: "radio-bridge",
          });
        }
```